priv/maker-ui/en.json

```diff
@@ -160,6 +160,7 @@
     "changing_role_owner_to_admin": "Changing your role to an Admin will make it so you can’t see billing info.",
     "changing_role_owner_to_contributor": "Changing your role to a Collaborator will make it so you can’t see billing info, or change others' roles.",
     "confirm_role_change": "Are you sure you want to change your role?",
+    "cancel": "Cancel",
```

Comment:
use `cancel_label`
we're trying to keep a pattern of "_label" for button labels